page-templates/evergreen.php

```diff
@@ -33,15 +34,46 @@ function add_body_classes_for_evergreen_page( $classes ) {
 
 add_filter( 'body_class', 'add_body_classes_for_evergreen_page' );
 
-$context                        = Timber::get_context();
-$post                           = new TimberPost();
-$page_meta_data                 = get_post_meta( $post->ID );
+$context        = Timber::get_context();
+$post           = new TimberPost();
+$page_meta_data = get_post_meta( $post->ID );
+
+// Retrieve P4 settings in order to check that we add only categories that are children of the Issues category.
+$options  = get_option( 'planet4_options' );
+$category = get_the_category( $post->ID )[0];
```

Comment:
Just a small detail. I think that this would return an empty array if the post does not belong to a category, resulting in a php warning.